src/chart/generateCategoricalChart.tsx

```diff
@@ -2244,42 +2244,45 @@ export const generateCategoricalChart = ({
       return null;
     }
 
+    renderMap = {
+      CartesianGrid: { handler: this.renderGrid, once: true },
+      ReferenceArea: { handler: this.renderReferenceElement },
+      ReferenceLine: { handler: this.renderReferenceElement },
+      ReferenceDot: { handler: this.renderReferenceElement },
+      XAxis: { handler: this.renderXAxis },
+      YAxis: { handler: this.renderYAxis },
+      Brush: { handler: this.renderBrush, once: true },
+      Bar: { handler: this.renderGraphicChild },
+      Line: { handler: this.renderGraphicChild },
+      Area: { handler: this.renderGraphicChild },
+      Radar: { handler: this.renderGraphicChild },
+      RadialBar: { handler: this.renderGraphicChild },
+      Scatter: { handler: this.renderGraphicChild },
+      Pie: { handler: this.renderGraphicChild },
+      Funnel: { handler: this.renderGraphicChild },
+      Tooltip: { handler: this.renderCursor, once: true },
+      PolarGrid: { handler: this.renderPolarGrid, once: true },
+      PolarAngleAxis: { handler: this.renderPolarAxis },
+      PolarRadiusAxis: { handler: this.renderPolarAxis },
+      Customized: { handler: this.renderCustomized },
+    };
+
+    surfaceStyle = { width: '100%', height: '100%' };
```

Comment:
As this is a pure constant, why not move it out completely, i.e. to the top of the file? 
Additionally I would prefer a name that describes intention as opposed to one that relates to the place of usage (as usage might change / evolve over time) 


```
const FULL_WIDTH_AND_HEIGHT = { width: '100%', height: '100%' };
```

Reply:
That's a good idea :smile: